### PR TITLE
Support call parameter

### PR DESCRIPTION
**Pull request by Silence-112**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added a parameter to select which unit calls for support (radioman or SL)

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

See if there is any difference in support calls when using SL parameter as opposed to Radioman

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/Stringtable.xml
Comment (wersal454) [inline A3A/addons/core/Stringtable.xml:1]: "<Key ID="STR_A3AU_radioman_support"
                <Original Support Caller</Original
                <English Support Caller</English
                <Russian помощь зов</Russian
            </Key

"<Russian>помощь зов</Russian>" probably needs to be changed depending on the context

Comment (Silence-112) [reply A3A/addons/core/Stringtable.xml:1]: Do you have a better one in mind?

Comment (wersal454) [reply A3A/addons/core/Stringtable.xml:1]: I need a context, like what sentence it can be used at or what is it actually for.

Comment (Silence-112) [reply A3A/addons/core/Stringtable.xml:1]: It's basically which person will call in reinforcements, idk how else to describe it

Comment (wersal454) [reply A3A/addons/core/Stringtable.xml:1]: well "помощь зов" can be translated as "help(not support)  call"

So I think "Вызывающий поддержку" is the right thing. It means (roughly) "The one who calls for support"

**Pull request merged**